Infrastructure/Persistence/EfCoreRepository/ApplicantRepository.cs

```diff
@@ -29,14 +29,9 @@ public Task<Applicant> GetApplicantAsync(Expression<Func<Applicant, bool>> expre
             throw new NotImplementedException();
         }
 
-        public bool IsExitByEmail(string email)
-        {
-            throw new NotImplementedException();
-        }
+        public bool IsExitByEmail(string email) => _context.Students.Any(s => s.EmailAddress == email);
```

Comment:
Why are you not using AnyAsync to make the method async

Reply:
it will be add sir